docs/reference/contributing/target/sai.md

```diff
@@ -62,6 +54,7 @@ If the device is a *transmitter*:
 A target must also define these elements to allow tests to be run:
 
 - `#define SAI_DEFAULT_SAMPLE_RATE (xxxxxU)`.
+  This macro is used by the tests to validate that the device behaves as expected. This parameter is device dependant, you need to set it to any value supported by the target.
```

Comment:
Please change to "The tests use this macro to validate that the device behaves as expected. This parameter is device dependent; you need to set it to any value that the target supports."